install/install.sh

```diff
@@ -137,6 +138,7 @@ cp .env.example .env
 sed -i.bak "s/RCTF_NAME=.*$/RCTF_NAME=$(echo "$RCTF_NAME"  | sed -e 's/\\/\\\\/g; s/\//\\\//g; s/&/\\\&/g')/g" .env
 sed -i.bak "s/RCTF_TOKEN_KEY=.*$/RCTF_TOKEN_KEY=$(echo "$RCTF_TOKEN_KEY"  | sed -e 's/\\/\\\\/g; s/\//\\\//g; s/&/\\\&/g')/g" .env
 
+chmod 600 .env .env.example
```

Comment:
Do we want to silently change permissions on `.env`? Might be good to alert the user

Reply:
ig we could alert the user. I didn't alert because it doesn't seem like a major change that the user needs to be aware of.

Reply:
Ah ok, I guess its a matter of personal preference. Feel free to merge then